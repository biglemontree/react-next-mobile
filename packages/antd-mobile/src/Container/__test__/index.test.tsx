import * as React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import { Container } from '..'

describe('Avatar', () => {
  it('snapshot', () => {
    const component = shallow(
      <div>
        <h3>Basic</h3>
        <Container
          title="花见花开"
          icon={{
            type: 'arrow',
          }}
        >
          消费信贷体验技术部
        </Container>
        <h3>Icon</h3>
        <Container
          title="人见人爱"
          icon={{
            type: 'close',
          }}
        >
          消费信贷体验技术部
        </Container>
        <h3>Thumb</h3>
        <Container
          thumb="https://gw.alipayobjects.com/zos/rmsportal/MRhHctKOineMbKAZslML.jpg"
          title="人见人爱"
          icon={{
            type: 'more',
          }}
        >
          消费信贷体验技术部
        </Container>
      </div>,
    )

    expect(toJSON(component)).toMatchSnapshot()
  })
})
