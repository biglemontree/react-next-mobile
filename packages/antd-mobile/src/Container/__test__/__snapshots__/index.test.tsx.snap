// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar snapshot 1`] = `
<div>
  <h3>
    Basic
  </h3>
  <Container
    icon={
      Object {
        "type": "arrow",
      }
    }
    title="花见花开"
  >
    消费信贷体验技术部
  </Container>
  <h3>
    Icon
  </h3>
  <Container
    icon={
      Object {
        "type": "close",
      }
    }
    title="人见人爱"
  >
    消费信贷体验技术部
  </Container>
  <h3>
    Thumb
  </h3>
  <Container
    icon={
      Object {
        "type": "more",
      }
    }
    thumb="https://gw.alipayobjects.com/zos/rmsportal/MRhHctKOineMbKAZslML.jpg"
    title="人见人爱"
  >
    消费信贷体验技术部
  </Container>
</div>
`;
